cornedbeef/build.gradle

```diff
@@ -10,10 +10,10 @@ android {
     buildToolsVersion "22.0.1"
 
     defaultConfig {
-        minSdkVersion 9
+        minSdkVersion 12
         targetSdkVersion 22
         versionCode 5
-        versionName "1.2.5"
+        versionName "1.2.6"
```

Comment:
Since we're bumping the SDK version this should probably be a major version bump